tests/integration/keystone.py

```diff
@@ -247,7 +247,7 @@ def test_admin_call(keystone_projects, user_info):
     sess = _get_keystone_session(username=user_info['name'],
                                  password=user_info['password'],
                                  project_name=user_info['project_name'])
-    resp = _do_get(sess, 'admin-only')
+    resp = _do_get(sess, 'v0/admin-only')
```

Comment:
wouldn't it make more sense, at least in terms of planning for the future, to make this change inside of `_do_get` itself, instead of editing each call to it?

Reply:
hmm, doing this would allow us to test different versions with different functions. And maybe some functions we want to label them as stable first.

Reply:
we can leave it as you have it